src/utils/cache.rs

```diff
@@ -186,7 +192,20 @@ fn pipeline_delete(con: &redis::Connection, keys: Iter<String>) {
     pipeline.execute(con);
 }
 
+fn scan_match_count<'a, P: ToRedisArgs, C: ToRedisArgs, RV: FromRedisValue>(
+    con: &'a redis::Connection,
+    pattern: P,
+    count: C,
+) -> redis::RedisResult<redis::Iter<'a, RV>> {
+    redis::cmd("INFO")
+        .cursor_arg(0)
+        .arg("MATCH")
+        .arg(pattern)
+        .arg("COUNT")
+        .arg(count)
+        .iter(con)
```

Comment:
you could assign this to `let count : usize = redis::cmd("INFO")....unwrap()` so you don't have to use the internal result type of redis. You can go as far as coercing the type to `Option<usize>` if you end the comand with `ok()`